apps/dashboard/src/app/app.component.ts

```diff
@@ -62,7 +63,8 @@ export class AppComponent implements AfterViewInit, OnInit, OnDestroy {
     if (this.config.getVersion()) {
       this.version = this.config.getVersion();
     }
-    this.checkStatusAlert();
+    this.checkMessages(true);
+    this.checkMessages(false);
```

Comment:
this is a bit weird and difficult to understand… why do we need to call the method twice with true and false? 
Could we have well named method doing these call instead so we can understand exactly what we're trying to achieve here?